OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -13,6 +13,10 @@ import RileyLinkBLEKit
 import UserNotifications
 import os.log
 
+fileprivate let bolusConfirmationBeeps: Bool = true      // whether to emit bolus confirmation beeps
```

Comment:
Please remove these vars. If we want to make various options available to users, they should be as UI elements; otherwise, lets pick a set that works for most people and use that.  The latter is what I'd prefer.  Based on feedback, it seems like Bolus, and suspend/resume, and basal change beeps fit that mark.

Reply:
All these variables are now removed but one. See additional comments on recent commit for details.